src/baseTrie.ts

```diff
@@ -398,11 +418,10 @@ export class Trie {
 
   /**
    * Walks a trie until finished.
-   * @method _walkTrie
    * @private
    * @param {Buffer} root
    * @param {Function} onNode - callback to call when a node is found
-   * @returns {Promise} - returns when finished walking trie
+   * @returns Returns when finished walking trie.
```

Comment:
```suggestion
   * @returns A promise when finished walking trie.
```

Reply:
well, it doesn’t return a promise when finished walking trie. the promise resolves when finished.

Reply:
My bad, thanks for the clarification!